pkg/cmd/repo/deploy-key/list/list.go

```diff
@@ -0,0 +1,125 @@
+package list
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/cli/cli/v2/internal/config"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/cli/cli/v2/utils"
+	"github.com/spf13/cobra"
+)
+
+type ListOptions struct {
+	IO         *iostreams.IOStreams
+	Config     func() (config.Config, error)
+	HTTPClient func() (*http.Client, error)
+	BaseRepo   func() (ghrepo.Interface, error)
+}
+
+func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Command {
+	opts := &ListOptions{
+		IO:         f.IOStreams,
+		Config:     f.Config,
+		HTTPClient: f.HttpClient,
+		BaseRepo:   f.BaseRepo,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List deploy keys in your GitHub repository",
```

Comment:
Nitpick:
```suggestion
		Short: "List deploy keys in the repository",
```
You can list deploy keys in any kind of repository (provided you have access). It doesn't have to be a repository that you own.